webapp/src/Controller/Jury/ConfigController.php

```diff
@@ -111,6 +117,9 @@ public function indexAction(EventLogService $eventLogService, Request $request):
                 'data' => $data
             ];
         }
+        if ($allData[5]['data'][1]['value']!==2) {
+            array_splice($allData[1]['data'],12,1);
```

Comment:
What...is...this...? 🤣

Reply:
This is a hard lock that we NEVER change the options... also it can be removed because of different implementation ;)